Woof/Woof/View/OwnerFlow/OwnerOnboardingView.swift

```diff
@@ -2,16 +2,7 @@ import SwiftUI
 
 /// A view to onboard the owner by filling in the mandatory information about themselves.
 struct OwnerOnboardingView: View {
-    /// View model responsible to manage data from model layer
-    @ObservedObject var viewModel = OwnerOnboardingViewModel()
-
-    /// A boolean indicating whether the save button should be enabled.
-    var isSaveButtonEnabled: Bool {
-        viewModel.name.isEmpty ||
-            viewModel.phone.isEmpty ||
-            viewModel.city.isEmpty ||
-            viewModel.address.isEmpty
-    }
+    typealias OwnerOnboardingViewModel = OwnerProfileViewModel
```

Comment:
Could we move it to private section, just to be consistent?

Reply:
I agree with you, the view model is in the private section. Fixed